torch/onnx/utils.py

```diff
@@ -154,7 +160,7 @@ def _model_to_graph(model, args, f, verbose=False, training=False,
         graph, torch_out = _trace_and_get_graph_from_model(model, args, training)
         params = list(_unique_state_dict(model).values())
 
-    graph = _optimize_graph(graph, aten)
+    graph = _optimize_graph(graph, aten, onnx_pass=not export_raw_ir)
```

Comment:
Can we call the argument `export_raw_ir` everywhere? Adding a different name is confusing